### DISCUSSION TITLE
context.decorator filename has typo error

### DISCUSSION THREAD

**Discussion by gecm0** [radical-sh/public-docs#20] [Issues]
![CleanShot 2024-03-08 at 11 59 41@2x](https://github.com/radical-sh/public-docs/assets/9404632/4e27b7b7-dbbf-423b-a45a-519423452cc3)

I expect `context` instead of `contex`

**Comment by Senthil-Sivanath** [comment]
Thanks for reporting.  Issue is fixed.
